theano/tensor/scipy_ndimage.py

```diff
@@ -0,0 +1,47 @@
+import theano
+from theano.gradient import DisconnectedType
+from scipy.ndimage.interpolation import zoom
+import numpy as np
+
+class Zoom(theano.Op):
+    # Properties attribute
+    __props__ = ()
+
+    def __init__(self):
+        super(Zoom, self).__init__()
+
+    def make_node(self, x, ref):
+        x = theano.tensor.as_tensor_variable(x)
+        ref = theano.tensor.as_tensor_variable(ref)
```

Comment:
Wouldn't a shape work here as well? You don't really use `ref` except to get its shape.